src/client/js/controller/welcome.js

```diff
@@ -63,32 +63,29 @@ class Welcome {
         this.view.help.setWelcomeContent(
           'FunCoNN (Beta)',
           [
-            '<b>Welcome to FunCoNN (Beta): Functional Connectivity on NemaNode</b>',
+            'Welcome to <b>FunCoNN (Beta): <u>Fun</u>ctional <u>Co</u>nnectivity on <u>N</u>ema <u>N</u>ode</b>',
             '<p>',
-            'This is an interactive browser of functional connectivity measurements of',
-            '<i>C. elegans</i> overlaid on select connectomics data using the software package',
-            '<b>NemaNode</b>.',
+            'This is an interactive browser of functional connectivity measurements of <i>C. elegans</i> overlaid',
+            'on select connectomics data using the <a href="https://nemanode.org/" target="_blank">Nema Node</a> software.',
             '<p>',
-            'Functional connectivity measurements show the casual direction of information',
-            'flow, polarity and functional strength of neural connections based on',
-            'optogenetic perturbations and calcium imaging.',
+            'Functional connectivity measurements show the causal direction of information flow, polarity',
+            'and functional strength of neural connections based on optogenetic perturbations and calcium',
+            'imaging. Functional measurements shown in this visualization are from a forthcoming',
+            'manuscript from the <a href="http://leiferlab.princeton.edu/" target="_blank">Leifer Lab</a>: Randi, Sharma, Dvali and Leifer, &#8220Direct measurements',
+            'of functional connectivity in C. elegans reveals contributions from extrasynaptic signaling,&#8221 in',
+            'prep. <b>Only those measured connections with a q-value less than 0.05 are shown.</b> The',
+            'absence of significance does not imply significance of absence.',
             '<p>',
-            'Functional measurements shown in this visualization are from a forthcoming',
-            '(in prep) manuscript from the Leifer Lab:',
+            'Measurements are displayed using <a href="https://nemanode.org/" target="_blank">Nema Node</a>, an open source <a href="https://github.com/zhenlab-ltri/NemaNode" target="_blank">software package</a> described',
+            'in <a href="https://doi.org/10.1038/s41586-021-03778-8" target="_blank">Witvleit et al., 2021</a> from the <a href="https://www.zhenlab.com/" target="_blank">Zhen</a>, <a href="https://scholar.harvard.edu/aravisamuel" target="_blank">Samuel</a>, and <a href="https://lichtmanlab.fas.harvard.edu" target="_blank">Lichtman</a> labs. Select connectomics',
+            'datasets from Witvleit et al, and others are also displayed.',
             '<p>',
-            'Randi, Dvali, Sharma & Leifer, <i>Direct Measurements of Functional Connectivity.</i>',
+            'This project is in beta and has not yet been peer reviewed. Data and code are released under',
```

Comment:
Another statement guaranteed to go stale - same suggestion.

Reply:
Let's not change any of the text that Dr. Leifer provided.

Reply:
I didn't suggest changing the the text.  I suggested inserting a comment and creating an issue.  But if we're confident we'll catch every update that needs to eventually be made, then NBD.

Reply:
Sorry, misread that. I do like creating an issue and noting the places we should change.

Reply:
Done, #98 and #99.